dotCMS/src/main/java/com/dotmarketing/factories/MultiTreeAPIImpl.java

```diff
@@ -254,6 +257,20 @@ public java.util.List<MultiTree> getMultiTrees(final String parentInode) throws
     @CloseDBIfOpened
     public List<MultiTree> getMultiTreesByVariant(final String parentId, final String variantName) throws DotDataException {
 
+        List<MultiTree> loadObjectResults = getMultiTreeByVariantWithoutFallback(parentId, variantName);
+
+        if (!UtilMethods.isSet(loadObjectResults)) {
+            if (!VariantAPI.DEFAULT_VARIANT.name().equals(variantName)) {
+                loadObjectResults = getMultiTreeByVariantWithoutFallback(parentId, VariantAPI.DEFAULT_VARIANT.name());
+            }
+        }
+
+        return loadObjectResults;
+    }
+
+    private List<MultiTree> getMultiTreeByVariantWithoutFallback(String parentId, String variantName)
```

Comment:
set to final

Reply:
done https://github.com/dotCMS/core/pull/23258/commits/be0dccb64465aa648a685a47be69411aa478f1b7#diff-926e9e9e22e2c282fa305bbe50022e93d9d91d6b83a5c33c8e29cacde4a0b52cR269